app/views/layouts/_footer.html.erb

```diff
@@ -25,6 +25,15 @@
     <p><%= t('layout._footer.different_licenses') %></p>
     <p><%= raw t('layout._footer.font_use', :url => 'http://www.theleagueofmoveabletype.com/junction') %></p>
     <p><%= raw t('layout._footer.server_donation', :url => 'http://rackspace.com/') %></p>
+    <ul class="dropdown">
+      <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+        <%= t('layout._footer.language.language_title') %> <b class="caret"></b>
+      </a>
+      <ul class="dropdown-menu">
+        <li><%= link_to t('layout._footer.language.english'), change_locale_path(:en) %></li>
```

Comment:
Can you reduce duplicated code here (and in the two matching sections) by iterating over a list of available languages stored or set at some central location?